ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayTasksSection.swift

```diff
@@ -11,23 +11,30 @@ import Backend
 
 struct TodayTasksSection: View {
     @EnvironmentObject private var collection: UserCollection
+    @Binding var sheet: Sheet.SheetType?
     
-    var isSunday: Bool {
-        Calendar.current.component(.weekday, from: Date()) == 1
+    private var tasksCount: Int {
+        collection.dailyCustomTasks.tasks.count
+    }
+    
+    private var rows: Int {
+        var rowsFloat = CGFloat(tasksCount) / 4.0
+        rowsFloat.round(.up)
+        return Int(rowsFloat)
     }
     
     // MARK: - Task Bubble
-    private func makeTaskBubble(icon: String, taskName: DailyTasks.taskName) -> some View {
-        var task: DailyTasks.Task {
-            collection.dailyTasks.tasks[taskName]!
+    private func makeTaskBubble(taskId: Int) -> some View {
+        var task: DailyCustomTasks.CustomTask {
+            collection.dailyCustomTasks.tasks[taskId]
         }
         
         return ZStack {
             Circle()
                 .foregroundColor(Color("ACBackground"))
-            Image(icon)
+            Image(task.icon)
                 .resizable()
-                .aspectRatio(taskName == DailyTasks.taskName.villagerHouses ? 0.8 : 1, contentMode: .fit)
+                .aspectRatio(1, contentMode: .fit)
```

Comment:
I'm not sure about this one, so double check, but if the aspect ratio is `1`, I think we could write directly
```swift
.aspectRatio(contentMode: .fit)
```

Reply:
Oh damn. This is an old part of code with the normal dailyTasks. Task name doesn’t exists in the new dailycustomtasks. I’ll remove this. It was a problem with the villagers icon that is bigger then the others.